flink-connectors/flink-connector-base/src/main/java/org/apache/flink/connector/base/source/reader/splitreader/SplitReader.java

```diff
@@ -33,7 +33,7 @@
  * @param <SplitT> the split type.
  */
 @PublicEvolving
-public interface SplitReader<E, SplitT extends SourceSplit> {
+public interface SplitReader<E, SplitT extends SourceSplit> extends AutoCloseable {
```

Comment:
The method itself can be removed from `SplitReader`as well: It doesn't add any value. Its documentation is generic as well.

Reply:
@XComp thanks. i have updated this pr.